### PR TITLE
patch-reset

### PR DESCRIPTION
**Pull request by MengZn**
We should reset IPVS tables do so manually when reset kubernetes cluster by kubeadm.

### CODE REVIEW

File: content/en/docs/setup/independent/create-cluster-kubeadm.md [hunk 1]
```diff
@@ -551,6 +551,12 @@ The reset process does not reset or clean up iptables rules or IPVS tables. If y
 iptables -F && iptables -t nat -F && iptables -t mangle -F && iptables -X
 ```
 
+If you wish to reset IPVS tables,you must do so manually to reset your system's IPVS tables:
```
Comment (neolit123) [inline content/en/docs/setup/independent/create-cluster-kubeadm.md]: please change to:
```
If you wish to reset IPVS tables, you must call the following command:
```

Comment (Rajakavitha1) [reply content/en/docs/setup/independent/create-cluster-kubeadm.md]: Suggested nit: To reset IPVS tables, call the following command:

**Comment by Hartmarken348** [comment]
Jump

File: content/en/docs/setup/independent/create-cluster-kubeadm.md (hunk 1 above)
Comment (MengZn) [reply content/en/docs/setup/independent/create-cluster-kubeadm.md]: @neolit123 @Rajakavitha1  Thank you so much.
```
If you wish to reset IPVS tables, you must call the following command:
```

Comment (MengZn) [reply content/en/docs/setup/independent/create-cluster-kubeadm.md]: @neolit123 I have corrected this description!

**Comment by MengZn** [comment]
@zacharysarah Can you give this PR a approve?

**Pull request merged**